extensions/markdown-language-features/src/util/topmostLineMonitor.ts

```diff
@@ -22,11 +35,50 @@ export class TopmostLineMonitor extends Disposable {
 				}
 			}
 		}));
+
+		this._register(vscode.window.onDidChangeActiveTextEditor(textEditor => {
+
+			// When at a markdown file, apply existing scroll settings from static preview if last editor was custom.
+			// Also save reference to text editor for line number reference later
+			if (textEditor && textEditor.document && isMarkdownFile(textEditor.document)) {
+				if (this.isPrevEditorCustom) {
+					const line = this.getPreviousStaticEditorLineByUri(textEditor.document.uri);
+					if (line) {
+						this._onEditorNeedsScrolling.fire({ line: line, editor: textEditor });
+					}
+				}
+				this.setPreviousMDTextEditorLine(textEditor);
+			}
+
+			this.isPrevEditorCustom = (textEditor === undefined);
```

Comment:
`isPrevCustomEditor` will up being true when you do anything that causes there not to be an active text editor, such as switching to an extension page or maximizing the panel

Reply:
Ok, will link `isPrevCustomEditor` to `PreviewManager._activePreview` == `undefined`